docs/src/modules/components/JoyStartingLinksCollection.js

```diff
@@ -28,7 +28,7 @@ const content = [
     title: 'Joy UI for Figma',
```

Comment:
Something like:

```suggestion
    title: 'Joy UI community file',
    description:
      'The free version of Joy UI components, with variables, variants, and states, in your favorite design tool.',
```

Reply:
I think we should mention Figma, though 😅 "Joy UI community file" is not immediately clear. We can mention "community" in the description, though!

Reply:
Isn't the Figma logo enough? 🤔

Reply:
It is! 😬 But in further thinking, not sure if the title should pinpoint the community file specifically — even though the link points to it, it's just a way to give folks access to the Figma file without any barriers. The paid file should ultimately be the end goal, particularly given the community file is somewhat shallower compared to the paid file.

Reply:
I think it works too. Besides, there is a lot of information about it on the community file description 👍